Add `The Followpoint: WhiteCat, the cat who dominated osu!` news post (#11721)
ppy/osu-wiki: news/2024/2024-07-15-the-followpoint-whitecat-the-cat-who-dominated-osu.md

---
layout: post

> title: "The Followpoint: WhiteCat, the cat who dominated osu!"

Reviewer: Just a suggestion

```suggestion
title: "The Followpoint: WhiteCat, The legendary cat who dominated osu!"
```

Reviewer 2: i don't think it's necessary (or in the correct casing)

date: 2024-07-15 17:00:00 +0000
series: The Followpoint
---

Dive into an interview with WhiteCat, a legendary *cat* and an osu! player!

![](/wiki/shared/news/2024-07-15-the-followpoint-whitecat-the-cat-who-dominated-osu/banner.jpg)

Hey, did you hear that? That's the sound of a new interview! Today we are joined by WhiteCat, a top player who many remember as someone who was dominating the scene before mrekk's rise as a former rank #1 player!

So, let's get right to reading!

---

*Hello, let's start with an introduction, how would you introduce yourself to people who don't know you?*

> > Hi, I'm WhiteCat, former nr.1 osu! player, now kinda retired.

Reviewer: ```suggestion
> Hi, I'm WhiteCat, former rank #1 osu! player, now kinda retired.
```

*How did you get introduced to osu! and what was your first impression of it?*

> I got introduced to osu! back when MCPvP disbanded, yet I really did not like it the first time I tried it. I layed off the game for roughly a year and then came back and stuck with it 

*I know many have already heard about it, but could you please tell us how did you come up with your username “WhiteCat”, for people who don’t know that?*

> The story begins with a neighborhood black cat who used to visit us. Took inspiration from her and wanted to name myself BlackCat. As this was taken though, I opted for WhiteCat.

*Since you have some experience with leading a healthy life, do you feel like there can be a big mental and physical help by visiting a gym?*

> I think there are numerous benefits to the going to the gym. I think this applies to a lot of types of exercise though, and in the end it ties back to moving your body a lot

*What was the most memorable moment for you in your osu! career?*

> > Most definitely the unban (WAS?)

Reviewer 2: will need @peppy's opinion on this one as it's about restrictions

Author: A lil update, due to style change it's now line 108

Reviewer 2: removing these instead in order to prevent delays in the schedule down the line

*What other rhythm games (or osu! game modes) besides osu! have you played?*

> It went like this: Geometry Dash - osu! mobile - normal osu!

*Who would you say are the best osu! player(s) at the moment and why?*

> I'm pretty out of loop currently, so I don't know a lot of the new top players. mrekk got that phenotype though.

*Is there any advice that you would give to people who are just starting out?*

> Enjoy game.

*Will you ever attend some IRL tournaments? Like The Roundtable for example.*

> Possibly, maybe even likely.

*What is your favorite score that you’ve ever set and which score would be your “dream” one to set in the future?*

> It is tough to choose one, yet my all time favorite might just be revolutionary etude, just because I could have never fathomed.

*Who would you want to meet IRL the most if you could meet any osu! player?*

> Vaxei, always been pleasant to chat with and turns out we share quite a few interests.

*To finish things up, what does the future hold for you? Do you have anything else that you'd like to mention?*

> Bike vlogs.

---

> Huge thanks to WhiteCat for joining us for an interview and answering these questions! And ofcourse, I hope you as readers enjoyed reading this interview! Please let us know what you think in the comments.

Reviewer 2: ```suggestion
Huge thanks to WhiteCat for joining us for an interview and answering these questions! And of course, I hope you as readers enjoyed reading this interview! Please let us know what you think in the comments.
```
